libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/types.ts

```diff
@@ -47,6 +60,50 @@ export const isGranteeUser = (obj: unknown): obj is IGranteeUser => {
     return !isEmpty(obj) && (obj as IGranteeUser).type === "user";
 };
 
+/**
+ * @internal
+ */
+export interface IGranularGranteeUser extends IGranteeBase {
```

Comment:
What is benefit to extend from `IGranteeBase` instead of `IGranteeUser`? Same for the group?

Reply:
None I believe, was not sure if values like `isCurrentUser`, `isOwner` would still apply for Panther, therefore was using the basic one. But I changed it to `IGranteeUser` & `IGranteeGroup`